front-end/src/App.js

```diff
@@ -1,16 +1,33 @@
-import './App.css';
-import Body from './components/Content/Body';
-import Footer from './components/Footer/Footer';
-import Header from './components/Header/Header';
-import Sidebar from './components/Sidebar/Sidebar';
+import "./App.css";
+import Post from "./components/Content/Post/Post";
+import User from "./components/Content/User/User";
+import Footer from "./components/Footer/Footer";
+import Header from "./components/Header/Header";
+import Breadcrumb from "./components/Content/Breadcrumb/Breadcrumb";
+import Sidebar from "./components/Sidebar/Sidebar";
+import Pagination from "./components/Content/Pagination/Pagination";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 
 function App() {
   return (
     <>
       <Header />
-      <div className='container'>
+      <div className="container">
         <Sidebar />
-        <Body />
+        <div className="sub-container">
+          <Breadcrumb />
```

Comment:
Good job because use help implement the routes heres.
I next lesson I will explain more the way it use it better.